sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -74,50 +62,52 @@ export class AttestationResponse<T> {
 }
 
 // @public
-export interface AttestationResult {
-    cnf?: any;
-    deprecatedEnclaveHeldData?: Uint8Array;
-    deprecatedEnclaveHeldData2?: Uint8Array;
-    deprecatedIsDebuggable?: boolean;
-    deprecatedMrEnclave?: string;
-    deprecatedMrSigner?: string;
-    deprecatedPolicyHash?: Uint8Array;
-    deprecatedPolicySigner?: JsonWebKey;
-    deprecatedProductId?: number;
-    deprecatedRpData?: string;
-    deprecatedSgxCollateral?: any;
-    deprecatedSvn?: number;
-    deprecatedTee?: string;
-    deprecatedVersion?: string;
-    enclaveHeldData?: Uint8Array;
-    exp?: number;
-    iat?: number;
-    inittimeClaims?: any;
-    isDebuggable?: boolean;
-    iss?: string;
-    jti?: string;
-    mrEnclave?: string;
-    mrSigner?: string;
-    nbf?: number;
-    nonce?: string;
-    policyClaims?: any;
-    policyHash?: Uint8Array;
-    policySigner?: JsonWebKey;
-    productId?: number;
-    runtimeClaims?: any;
-    sgxCollateral?: any;
-    svn?: number;
-    verifierType?: string;
-    version?: string;
-}
+export class AttestationResult {
+    constructor(params: {
+        issuer: string;
+        version: string;
+        nonce?: string;
+        uniqueId: string;
+        runtimeClaims?: any;
+        inittimeClaims?: any;
+        policyClaims?: any;
+        verifierType: string;
+        policySigner?: AttestationSigner;
+        policyHash: Uint8Array;
+        isDebuggable?: boolean;
+        productId?: number;
+        mrEnclave?: string;
+        mrSigner?: string;
+        svn?: number;
+        enclaveHeldData?: Uint8Array;
+        sgxCollateral?: any;
+    });
+    get enclaveHeldData(): Uint8Array | undefined;
```

Comment:
@xirzec is using accessors still frowned upon?

Reply:
Is there another way of expressing a read only property in TS that doesn't involve property accessors?